source/_components/binary_sensor.raspihats.markdown

```diff
@@ -35,16 +35,45 @@ binary_sensor:
             name: PIR Bedroom
 ```
 
-Configuration variables:
-
-- **i2c_hats** (*Optional*): Array of used I2C-HATs.
-  - **board** (*Required*): The board name [Di16, Di6Rly6, DI16ac, DI6acDQ6rly].
-  - **address** (*Required*): The board I2C address, hex value.
-    - **channels** (*Required*): Array of used digital input channels.
-      - **index** (*Required*): Digital input channel index.
-      - **name** (*Required*): Friendly name to use for the frontend.
-      - **invert_logic** (*Optional*): Inverts the input logic, default is `false`.
-      - **device_class** (*Optional*): See device classes in [binary_sensor component](/components/binary_sensor/), default is `None`
+{% configuration %}
+i2c_hats:
+  description: An array of used I2C-HATs.
+  required: false
+  type: list
+  keys:
+    board:
+      description: The board name either Di16, Di6Rly6, DI16ac or DI6acDQ6rly.
+      required: true
+      type: string
+    address:
+      description: The board I2C address as HEX value.
+      required: true
+      type: string
+      keys:
+        channels:
```

Comment:
Since `channel` is not a key for `address`: fix the indentation.
(remove 2 spaces from each of the following lines)